ch02.adoc

```diff
@@ -268,3 +268,217 @@ They may not be attached to a group, even if all variables within that group use
 If attributes are present within groups without being attached to a variable, these attributes apply to the group where they are defined, and to that group's descendants, but not to ancestor or sibling groups.
 If a group attribute is defined in a parent group, and one of the child group redefines the same attribute, the definition within the child group applies for the child and all of its descendants.
 
+[[aggregation-variables, Section 2.8, "Aggregation Variables"]]
+=== Aggregation Variables
+
+An __aggregation variable__ is a variable which has been formed by combining (i.e. aggregating) multiple __fragments__ stored in __fragment files__ that are external to the file containing the aggregation variable, i.e. the __aggregation file__.
+A fragment is an array of data with sufficient metadata for it to be correctly interpreted in the context of the aggregation, as described by <<fragment-interpretation>>.
+The aggregation variable does not contain any actual data, instead it contains instructions on how to create its __aggregated data__ as an aggregation of the data from each fragment.
+
+Aggregation provides the utility of being able to view, as a single entity, a dataset that has been partitioned across multiple files, whilst taking up very little space on disk (since the aggregation file contains no copies of the data in the fragments).
+The fragment files may be CF-compliant or have any other format, thereby allowing an aggregation variable to act as CF-compliant view of non-CF datasets.
+Storing aggregations is useful for data analysis, as it avoids the computational expense of deriving the aggregation at the time of analysis; and for archive curation, as the aggregation can act as a metadata-rich archive index.
+
+An aggregation variable must be a scalar (i.e. it has no dimensions).
+It acts as a container for all of the usual attributes that describe the data, with the addition of two special attributes: one that defines the _aggregated dimensions_, i.e. the dimensions of the aggregated data; and one that provides the instructions on how the aggregated data is to be created.
+The data type of the aggregation variable must be the data type of the aggregated data, but the value of the aggregation variable's single element is immaterial.
+
+Aggregation variables may be used as any kind of variable (data variable, coordinate variable, cell measures variable, grid mapping variable, etc.), but it is recommended that container variables whose data are immaterial (such as grid mapping variables) are not encoded as aggregation variables.
```

Comment:
```suggestion
Aggregation variables may be used as any kind of variable (data variable, coordinate variable, cell measures variable, etc.), but it is recommended that container variables whose data are immaterial (such as grid mapping variables) are not encoded as aggregation variables.
```

Best not to list the one we go on to advise against using as a possibility first, in my opinion.

Reply:
It still makes sense as-is, though, so no blocker.